src/tutorials/0008-merkle-dags/03.md

```diff
@@ -0,0 +1,67 @@
+---
+    title: "Directed Acyclic Graphs (DAGs)"
+    type: "multiple-choice"
+---
+
+Before we can create content-addressable data structures, we have
+to define those structures in a way that enables us to talk about
+them precisely. For this purpose, we turn to graphs.
+
+A *graph* is a mathematical abstraction that is used to represent
+relationships among a collection of objects. It is common to use
+the term *node* to refer to an object in a graph, and the term *edge*
+to refer to a relation among objects. Most commonly, graphs are
+used to represent pairwise relationships between objects. For
+example, a graph might be used to map the roads connecting pairs
+of cities, or the friendships between students in a school.
+
+The example file hierarchy we introduced in the previous lesson
+also forms a graph (the directories and files act as our nodes,
+and the relationship between a directory and the files it
+contains gives us our edges). We’ve depicted it again in the
+figure below:
+
+![An image depicting a graph of the file directory described in Lesson Two, using simplified filenames. Arrows are drawn pointing from directories to their contents to indicate containment.](/tutorial-assets/TOOO8L03-directory-graph.png)
```

Comment:
The images prefixes are `TOOO8` instead of `T0008`.
So the paths and the files need to be renamed.

Reply:
Fixed!